keda/templates/13-keda-poddisruptionbudget.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if .Values.podDisruptionBudget }}
+{{- if .Values.podDisruptionBudget}}
```

Comment:
This should be removed in favor of the one below this line
```suggestion
```

Reply:
done